DED-258 adding sample apps links to auth documentation (#1210)
livechat/livechat-public-docs: src/pages/authorization/index.mdx

  allowfullscreen
></iframe>

# Sample Authorization Apps

> We've prepared two apps which include fully implemented authorization code samples. You can test them, experiment with them, or modify one of this app’s code and extend its functionality.

Reviewer: ```suggestion
We've prepared two apps with fully implemented authorization flows. You can test them, experiment with them, or modify their code and extend their functionality.
```

> 

Reviewer: - <a href="https://github.com/livechat/sample-app-popup-auth" target="_blank">Sample app with the popup flow</a>
- <a href="https://github.com/livechat/sample-app-redirect-auth" target="_blank">Sample app with the redirect flow</a>

Reviewer: Or maybe we could use the `<SectionLink>` component here? 👆🏻

Author: Tomorrow, I will see how it looks and decide:) However, it seems like a very good idea

Author: I have added normal links, as the `<SecrionLink>' component was not working when I tried. However, it still looks very similar to the `<SectionLink>' :)

Reviewer: ok

> Currently, two flows for obtaining an access token from the [LiveChat Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk) are available: <a href="https://github.com/livechat/sample-app-popup-auth" target="_blank">popup flow</a> and <a href="https://github.com/livechat/sample-app-redirect-auth" target="_blank">redirect flow</a>. Please check out our Sample authorization app tutorial if you want more information about our sample apps.

Reviewer: ```suggestion
The apps demonstrate how to implement two flows from the [LiveChat Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk): the [popup flow](/authorization/sign-in-with-livechat#popup-1) and the [redirect flow](/authorization/sign-in-with-livechat#redirect-1).

Please check out our Sample authorization app tutorial if you want more information about our sample apps.
```

  allowfullscreen
></iframe>

# Sample Authorization Apps

We've prepared two apps with fully implemented authorization flows. You can test them, experiment with them, or modify their code and extend their functionality.

The apps demonstrate how to implement two flows from the [LiveChat Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk): the [popup flow](/authorization/sign-in-with-livechat#popup-1) and the [redirect flow](/authorization/sign-in-with-livechat#redirect-1).

> Please check out our Sample authorization app tutorial if you want more information about our sample apps.

Author: Do we already know what the link here will be?

Reviewer: https://developers.livechat.com/updates/learn-authorization-with-sample-apps

  allowfullscreen
></iframe>

> # Sample Authorization Apps

Reviewer: ```suggestion
# Sample authorization apps
```

Reviewer: other headlines in this doc are sentence case

We've prepared two apps with fully implemented authorization flows. You can test them, experiment with them, or modify their code and extend their functionality:

- <a href="https://github.com/livechat/sample-app-popup-auth" target="_blank">Sample app with the popup flow</a>

- <a href="https://github.com/livechat/sample-app-redirect-auth" target="_blank">Sample app with the redirect flow</a>

The apps demonstrate how to implement two flows from the [LiveChat Accounts SDK](/authorization/sign-in-with-livechat#accounts-sdk): the [popup flow](/authorization/sign-in-with-livechat#popup-1) and the [redirect flow](/authorization/sign-in-with-livechat#redirect-1).

> Please check out our <a href="https://developers.livechat.com/updates/learn-authorization-with-sample-apps" target="_blank">Sample authorization app tutorial</a> if you want more information about our sample apps.

Reviewer: ```suggestion
To learn more about the sample apps and implementing Accounts SDK, read our <a href="https://developers.livechat.com/updates/learn-authorization-with-sample-apps" target="_blank">blog post</a>.
```
